_shared_content/develop/guides/formats/best_practices/authentications.md

```diff
@@ -0,0 +1,520 @@
+  "Hostname": "V-FOO",
+  "Keywords": -9214364837600034816,
+  "EventType": "AUDIT_SUCCESS",
+  "SeverityValue": 2,
+  "Severity": "INFO",
+  "EventID": 4624,
+  "SourceName": "Microsoft-Windows-Security-Auditing",
+  "ProviderGuid": "{54849625-5478-4994-A5BA-3E3B0328C30D}",
+  "Version": 1,
+  "Task": 12544,
+  "OpcodeValue": 0,
+  "RecordNumber": 10457874880,
+  "ProcessID": 744,
+  "ThreadID": 2352,
+  "Channel": "Security",
+  "Category": "Logon",
+  "Opcode": "Info",
+  "SubjectUserSid": "S-1-0-0",
+  "SubjectUserName": "-",
+  "SubjectDomainName": "-",
+  "SubjectLogonId": "0x0",
+  "TargetUserSid": "S-1-5-21-1574594750-1263408776-2012955550-69701",
+  "TargetUserName": "SVC_DD_SP-SEARCH",
+  "TargetDomainName": "KEY",
+  "TargetLogonId": "0xfbee0744",
+  "LogonType": "3",
+  "LogonProcessName": "NtLmSsp ",
+  "AuthenticationPackageName": "NTLM",
+  "WorkstationName": "V-FOO",
+  "LogonGuid": "{00000000-0000-0000-0000-000000000000}",
+  "TransmittedServices": "-",
+  "LmPackageName": "NTLM V2",
+  "KeyLength": "128",
+  "ProcessName": "-",
+  "IpAddress": "-",
+  "IpPort": "-",
+  "ImpersonationLevel": "%%1833",
+  "EventReceivedTime": "2010-06-18 15:28:24",
+  "SourceModuleName": "in",
+  "SourceModuleType": "im_msvistalog"
+}
+```
+
+The previous event will result into this ECS document:
+
+```json
+{
+  "event": {
+    "code": "4624",
+    "provider": "Microsoft-Windows-Security-Auditing",
+    "category": [
+      "authentication"
+    ],
+    "type": [
+      "start"
+    ],
+    "action": "authentication_network"
+  },
+  "sekoiaio": {
+    "client": {
+      "os": {
+        "type": "windows"
+      },
+      "name": "V-FOO"
+    },
+    "server": {
+      "name": "V-FOO",
+      "os": {
+        "type": "windows"
+      }
+    }
+  },
+  "action": {
+    "outcome": "success"
+  },
+  "host": {
+    "name": "V-FOO"
+  },
+  "process": {
+    "thread": {
+      "id": 2352
+    },
+    "pid": 744,
+    "id": 744,
+    "name": "NtLmSsp "
+  },
+  "user": {
+    "id": "S-1-0-0",
+    "target": {
+      "name": "SVC_DD_SP-SEARCH",
+      "domain": "KEY",
+      "id": "S-1-5-21-1574594750-1263408776-2012955550-69701"
+    },
+    "name": "-",
+    "domain": "-"
+  }
+}
+```
+
+With the following smart-description:
+
+```json
+{
+  "value": "{user.target.domain}\\{user.target.name} logged on to {sekoiaio.server.name} ({event.action})",
+  "relationships": [
+    {
+      "source": "user.target.name",
+      "target": "sekoiaio.server.name",
+      "type": "event.action"
+    }
+  ],
+  "conditions": [
+    {"field": "event.category", "value": "authentication"},
+    {"field": "event.type", "value": "start"},
+    {"field": "action.outcome", "value": "success"},
+    {"field": "event.action"},
+    {"field": "user.target.name"},
+    {"field": "user.target.domain"},
+    {"field": "sekoiaio.server.name"},
+    {"field": "event.provider", "value": "Microsoft-Windows-Security-Auditing"}
+  ]
+}
+```
+
+will result into the description:
```

Comment:
```suggestion
Will result into the description:
```
I am not sure that you can start a sentence by will if its not a question https://www.quora.com/Can-you-start-a-sentence-with-the-word-will.